meta-resin-common/recipes-connectivity/dnsmasq/files/dnsmasq.conf.systemd

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Wants=docker.service resin-net-config.service
-After=docker.service resin-net-config.service
+Requires=etc-dnsmasq.conf
```

Comment:
Shouldn't this read: etc-dnsmasq.mount instead? And maybe it would be a good idea to make it actually depend on connman.service as it is that service which ensures that dns entries are written into /etc/dnsmasq/resolv.conf